include/volk/volk_neon_intrinsics.h

```diff
@@ -147,4 +181,98 @@ static inline float32x4_t _vlogq_f32(float32x4_t x)
     return poly;
 }
 
+/* Evaluation of 4 sines & cosines at once.
```

Comment:
I'd like to start restructuring this file, starting with this PR:

When we add new code with a new license, let's append the whole piece of code including the license, inside a delineation comment such as:
```
/******************************************************************/
```
In this manner, we make it clear which sections of code are related to which license.

Once this PR is in place, I'll go through this file & rearrange it for how the code was added historically, keeping the licenses and related codes together.

Reply:
Ok sorry about the long delay, lots of things going on.

I will fix this now but rereading your comment I'm not 100% sure how you would like it. Do you want me to move the license down to right above the relevant function and enclose that whole block in a delineation comment? Then I will also have to refactor the MIT licensed code.

Reply:
This: Move the license down to right above the relevant function and enclose that whole block in a delineation comment. Yes that means refactoring a little. Hoping that won't be too difficult.

Reply:
@jdemel  and I discussed this briefly. My goal was to make it extremely clear which license/code went together; since the function is named in the license, it's not so critical to keep them together. Given that there's already another license / code combo split here (and probably elsewhere), and that the function is named for both licenses: I'm removing this request.